toggles/webapp/toggles.ts

```diff
@@ -62,6 +62,13 @@ const toggles = {
       description:
         'New search page to help develop new components and functionalities',
     },
+    {
+      id: 'searchPageEventsExhibitions',
+      title: 'Search page: Events & Exhibitions',
+      initialValue: false,
+      description:
+        "Events & Exhibitions will not go out at the same time as the rest of the new Search page, therefore we're adding another level of toggle within it to release them separately",
```

Comment:
This is more a commit message than a toggle description (and it's a bit wordy) – it makes sense if you know the development history of this feature, but not otherwise.

You're also repeating the explanation of toggles ("to release them separately").

I would go for something like:

> Include events and exhibitions on the new search page